gluon/_compat.py

```diff
@@ -0,0 +1,106 @@
+import sys
```

Comment:
Why are we duplicate this file? 

https://github.com/web2py/pydal/blob/b5f91ba4837da33f1f3036fa5a821a28dee68693/pydal/_compat.py